readthedocs/core/views.py

```diff
@@ -235,21 +237,30 @@ def github_build(request):
         except:
             # Generic post-commit hook
             obj = json.loads(request.body)
-        url = obj['repository']['url']
-        ghetto_url = url.replace('http://', '').replace('https://', '')
+        repo_url = obj['repository']['url']
+        hacked_repo_url = repo_url.replace('http://', '').replace('https://', '')
+        ssh_url = obj['repository']['ssh_url']
+        hacked_ssh_url = ssh_url.replace('git@', '').replace('.git', '')
```

Comment:
This really should be a regex, there are a number of cases this will fail for. I'm +1 on opening another ticket and dealing with it there.